integration-tests/deployment/devenv/jd.go

```diff
@@ -0,0 +1,71 @@
+package devenv
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	csav1 "github.com/smartcontractkit/chainlink/integration-tests/deployment/jd/csa/v1"
+	jobv1 "github.com/smartcontractkit/chainlink/integration-tests/deployment/jd/job/v1"
+	nodev1 "github.com/smartcontractkit/chainlink/integration-tests/deployment/jd/node/v1"
+)
+
+type JDConfig struct {
+	GRPC  string
+	WSRPC string
+	creds credentials.TransportCredentials
+}
+
+func NewClientConnection(cfg JDConfig) (*grpc.ClientConn, error) {
```

Comment:
NewClientConnection to what? If this is connecting to JD, lets add that in the func name. The package is just `devenv` so it doesn't give me much info as to what kind of client this is creating.